### PR TITLE
Add scroll to issue sidebar dropdowns

### LINKED ISSUE #31589: Add scroll to dropdowns
### Description

If there're too many option in dropdowns, ui kinda breaks

### Screenshots

image

### Gitea Version

 1.23.0+dev-300-g5791a73e75

### Can you reproduce the bug on the Gitea demo site?

Yes

### Operating System

_No response_

### Browser Version

latest chrome

### PR DESCRIPTION
**Pull request by a1012112796**
as title, fix #31589

examples:
屏幕截图 2024-07-09 110007  
---------------------
屏幕截图 2024-07-09 105146     
-------------------------
屏幕截图 2024-07-09 102303   
--------------------------
屏幕截图 2024-07-09 101441   
-----------------------
屏幕截图 2024-07-09 100859

### CODE REVIEW

File: web_src/js/features/user-auth-webauthn.ts [hunk 1]
```diff
@@ -174,6 +174,10 @@ async function webauthnRegistered(newCredential) {
 function webAuthnError(errorType, message) {
   const elErrorMsg = document.querySelector(`#webauthn-error-msg`);
 
+  if (!elErrorMsg) {
```
Comment (6543) [inline web_src/js/features/user-auth-webauthn.ts:177]: looks unrelated could you move it into another pull?

Comment (a1012112796) [reply web_src/js/features/user-auth-webauthn.ts:177]: just a small nit, looks not necessary to create a new pull request.

Comment (6543) [reply web_src/js/features/user-auth-webauthn.ts:177]: I prefere pulls per topic ... but i see your point ...

**Review by 6543 — APPROVED** [review]

**Comment by lunny** [comment]
It's better to use Vue rather than Go template there if you would like to do a refactor. Because they should be refactored to an instance UI in the future. So we don't need to do it twice.

**Comment by wxiaoguang** [comment]
1. It could only use tmpl at the moment, but not Vue
2. Some changes don't seem proper. For example: why using `no-select`? It has other meanings in code, and shouldn't be used as a new purpose.
3. I guess you mean `const $menu = $(`${select_id} > .menu`)`, but not use more `:not` filters.

**Comment by wxiaoguang** [comment]
4. Be careful of conflicts with https://github.com/go-gitea/gitea/pull/31083 , do not lose changes from it

**Comment by wxiaoguang** [comment]
This change doesn't seem good enough.

I have refactored the sidebar dropdowns, please follow the new "combo dropdown list" to add new enhancements to them.

 Refactor sidebar label selector #32460
